jvm/sargon-android/src/main/java/com/radixdlt/sargon/samples/NonFungibleLocalIdSample.kt

```diff
@@ -0,0 +1,22 @@
+package com.radixdlt.sargon.samples
+
+import androidx.annotation.VisibleForTesting
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import com.radixdlt.sargon.NonFungibleLocalId
+import com.radixdlt.sargon.newNonFungibleLocalIdInt
+import com.radixdlt.sargon.newNonFungibleLocalIdString
+
+
+@VisibleForTesting
+val NonFungibleLocalId.Companion.sample: Sample<NonFungibleLocalId>
+    get() = object : Sample<NonFungibleLocalId> {
+
+        override fun invoke(): NonFungibleLocalId = newNonFungibleLocalIdInt(value = 1337.toULong())
```

Comment:
and after adding `_sample() -> NonFungibleLocalId` UniFFI export, change this to use those ofc :)